internal/report/report.go

```diff
@@ -98,7 +120,26 @@ func (r *PolicyReport) AddResult(policy policiesv1.Policy, resource unstructured
 		).Msg("added result to report")
 }
 
-func (r *ClusterPolicyReport) AddResult(policy policiesv1.Policy, resource unstructured.Unstructured, auditResponse *admv1.AdmissionReview, responseErr error) {
+func (r *PolicyReport) GetSummary() (string, error) {
+	marshaled, err := json.Marshal(r.Summary)
```

Comment:
Just out of curiosity, why do we need to marshal that to JSON? Reading the method name I would not expect to get back a JSON encoded string.

I think we should write some inline docs explaining that and, eventually, rename the method to be something like `GetSummaryJSON` (although I'm not super happy/convinced about this name)

Reply:
This function is only used (so far) in a log.Debug(), which itself is a json output:
```
{"level":"info","dict":{"report name":"polr-ns-default","report ns":"default","summary":"{\"pass\":0,\"fail\":0,\"warn\":0,\"error\":0,\"skip\":0}"},"time":"2023-06-07T17:57:25+02:00","message":"updated PolicyReport"}
```

I have renamed it and added godocs in https://github.com/kubewarden/audit-scanner/pull/47/commits/09c651a5726c3a76fd0f184bdb90fbef3151f7c2.